bin/oli/Cargo.toml

```diff
@@ -34,6 +34,10 @@ anyhow = { version = "1.0" }
 clap = { version = "4.5", features = ["cargo", "string", "derive"] }
 dirs = { version = "5.0" }
 futures = { version = "0.3" }
+humansize = { version = "2.1" }
+humantime = { version = "2.1" }
+humantime-serde = { version = "1.1" }
+jiff = { version = "0.1.15" }
```

Comment:
opendal core already uses `chrono`, maybe we can use the same crate instead.

Reply:
I'll try. The major usage here I'd like to impl is format the current time as `YYYYMMDD` so that we can debug the bench target. But perhaps we can omit such a requirement first but instead add an option for whether delete the object on delete.